ifStatement.js

```diff
@@ -59,7 +88,16 @@ Steps:
     - If the user is younger than 18, the program should calculate how many years are left until they turn 18 and print the message "You will be eligible to vote in X years", where X is the number of years left.
 ************************************************************************************************/
 // TODO: ADD YOUR CODE BELOW
-
+let Age=prompt("ENTRE AGE ");
+if (Age >=18)
+{
+  console.log("Great! You are all set to vote.")
+}
+else if (Age < 18) {
+  console.log("You still have time to register to vote")
+}
+let younger=18-Age;
```

Comment:
This should be within an else statement, not outside it, as it will print regardless of whether the user is younger than 18 or not.
````
else {
     let younger=18-Age;
     ....
     }
```